### ISSUE TITLE
oneType' object has no attribute 'get_import_manager

### ISSUE THREAD

**Issue by dong9205** [cobbler/cobbler#2145]
I reported an error while executing the cobbler import~~~~~

````
task started: 2019-09-13_014521_import
task started (id=Media import, time=Fri Sep 13 01:45:21 2019)
running python triggers from /var/lib/cobbler/triggers/task/import/pre/*
running shell triggers from /var/lib/cobbler/triggers/task/import/pre/*
shell triggers finished successfully
Exception occured: <class 'AttributeError'>
Exception value: 'NoneType' object has no attribute 'get_import_manager'
Exception Info:
  File "/usr/local/python3/lib/python3.6/site-packages/cobbler/remote.py", line 84, in run
    rc = self._run(self)

  File "/usr/local/python3/lib/python3.6/site-packages/cobbler/remote.py", line 231, in runner
    self.logger

  File "/usr/local/python3/lib/python3.6/site-packages/cobbler/api.py", line 861, in import_tree
    import_module = self.get_module_by_name("manage_import_signatures").get_import_manager(self._collection_mgr, logger)

!!! TASK FAILED !!!
````

**Comment by rbberger** [comment]
Fixed by #2172

**Comment by SchoolGuy** [comment]
Then let's close this one.
